util/template/expression_template.go

```diff
@@ -0,0 +1,47 @@
+package template
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/Masterminds/sprig"
+	"github.com/antonmedv/expr"
+	"github.com/doublerebel/bellows"
+)
+
+const kindExpression kind = "="
+
+func init() {
+	if os.Getenv("EXPRESSION_TEMPLATES") != "false" {
+		kinds = append(kinds, kindExpression)
```

Comment:
Perhaps a function `registerKind(kindExpression)` might make the intent of this code more clear? The function could live in `kind.go`

Reply:
I like this, will implement